spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/SparkActionOptions.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.spark.actions;
+
+public class SparkActionOptions {
+
+  private SparkActionOptions() {
+  }
+
+  public static final String SKIP_ON_ERROR = "skip_on_error";
```

Comment:
While I think it makes sense to have these final constants, they only apply to Snapshot and Migrate, and I think this may not be the right place for them then? Or they need a java doc saying that this only applies to those particular actions.

Reply:
Have been added the javadoc

I introduce it here to migrate all Spark action options, such as the following:
https://github.com/apache/iceberg/blob/8155e5e8129365ed8d33b8cced2e851734a302c5/spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteReachableFilesSparkAction.java#L56-L57

https://github.com/apache/iceberg/blob/8155e5e8129365ed8d33b8cced2e851734a302c5/spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/ExpireSnapshotsSparkAction.java#L71-L72

https://github.com/apache/iceberg/blob/8155e5e8129365ed8d33b8cced2e851734a302c5/spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteManifestsSparkAction.java#L86-L87

If we don't think it makes sense, I can remove it